SwiftStyleGuide.swift

```diff
@@ -104,6 +104,24 @@ private extension MyStruct {
                             : MyCustomError.networkError
     }
 
+    func methodThatNeedsToCaptureSelf() {
+        // great guide on when and when not to capture self strongly
+        // https://medium.com/flawless-app-stories/you-dont-always-need-weak-self-a778bec505ef
+
+        // no need to explicitly capture self if you need a strong reference
+        foo.methodThatNeedsStrongCapture() {
+            // ...
+        }
+
+        // but of course we do add it for weak references
+        foo.methodThatNeedsWeakCapture() { [weak self] in
+            guard let `self` = self else { return }
```

Comment:
I _think_ I recall seeing somewhere that the backticks aren't needed anymore, but I'm not sure

Reply:
yep, you're right, the backticks are not needed anymore, `guard let self = self else { ... }` works fine 👍🏼

Reply:
awesome, thanks! I'll update accordingly